.github/pull_request_template.md

```diff
@@ -1,17 +1,8 @@
-### What does this PR do?
+### What this PR does / why we need it
 
-(Please set a descriptive PR title. Use this space for additional explanations.)
+<!-- Please mention any GitHub issue, set a descriptive PR title, and use this space for additional explanations. -->
 
-### What does it look like?
+### Checklist
 
-(If this is more than a textual change, a screenshot can help in some cases to speed up the review process.)
-
-### Any background context you can provide?
-
-(Please link public issues or summarize if not public.)
-
-### What is needed from the reviewers?
-
-### Have you maintained the front matter?
-
-(Please bump the last_review_date in case this qualifies as a review for an entire page. Provide user_questions which should be answered in the page. Provide a meaningful description.)
+- [ ] (Content changes only) Ran `make dev` to test content changes locally.
```

Comment:
This looks like we expect everybody to test changes locally. Do we do that?

I wouldn't recommend putting something optional into the checklist.

Reply:
IMO it is a good reminder in case you have modified a content file (which are the 95% of the cases)